index.html

```diff
@@ -0,0 +1,130 @@
+</br>
+</p>
+<p
+    
+    class="headline-primary-text-1"
+>I’m a software developer! I reside in Nairobi, Kenya. I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<div
+    
+    class="headline-social-media"
+><div
+    
+    class="headline-supporting-text"
+><p
+    
+    class="headline-letsconnect"
+>LET’S CONNECT</p></div></div>
+<div
+    
+    class=" headline-social-media-icons"
+><svg
+    width="375px"
+height="44px"
+preserve-aspect-ratio="none"
+view-box="0 0 171 20"
+fill="none"
+xmlns="http://www./2000/svw3.orgg"
+    
+><path
+    d="M 162.548 20 C 167.244 20 170.167 16.539 170.167 12.074 C 170.167 10.391 169.893 8.871 167.982 8.414 C 168.447 7.141 169.788 3.458 169.788 2.278 C 169.788 1.238 169.163 0.368 168.049 0.368 C 166.273 0.368 164.682 5.512 164.183 6.738 C 163.798 5.625 161.991 0 160.398 0 C 159.161 0 158.58 0.894 158.58 2.019 C 158.58 3.398 159.941 6.973 160.434 8.348 C 160.184 8.258 159.913 8.179 159.638 8.179 C 158.706 8.179 157.715 9.316 157.715 10.234 C 157.715 10.582 157.91 11.07 158.033 11.394 C 156.565 11.785 156 12.746 156 14.195 C 155.996 17.015 158.719 20 162.548 20 Z M 166.573 4.219 C 166.733 3.738 167.413 1.708 168.049 1.708 C 168.392 1.708 168.483 2.055 168.483 2.333 C 168.483 3.078 166.948 7.2 166.608 8.113 L 165.255 7.879 L 166.573 4.219 Z M 159.83 1.888 V 1.887 C 159.83 1.422 160.408 0.102 161.673 3.727 L 163.05 7.645 C 162.429 7.594 161.948 7.528 161.642 7.7 C 161.208 6.575 159.83 3.024 159.83 1.888 Z M 159.738 9.531 L 159.739 9.532 C 160.905 9.532 162.41 13.227 162.41 13.727 C 162.41 13.926 162.215 14.172 161.988 14.172 C 161.156 14.172 158.927 11.168 158.927 10.355 C 158.93 10.054 159.432 9.531 159.738 9.531 Z M 158.398 12.589 C 158.852 12.589 160.559 14.945 160.559 15.445 C 160.559 15.637 160.253 15.769 160.094 15.769 C 159.453 15.769 159.203 15.163 158.061 13.762 C 156.878 14.922 158.877 17.156 160.382 17.156 C 161.42 17.156 162.097 16.211 161.894 15.515 C 162.042 15.515 162.224 15.527 162.36 15.492 C 162.404 16.55 162.723 17.812 164.02 17.902 C 164.02 17.867 164.1 17.624 164.1 17.613 C 164.1 16.933 163.678 16.339 163.678 15.648 C 163.678 14.542 164.542 13.472 165.418 12.847 C 165.736 12.613 166.123 12.468 166.496 12.335 C 166.883 12.191 167.293 12.023 167.586 11.733 C 167.542 11.296 167.359 10.909 166.913 10.909 C 165.811 10.909 162.113 11.066 162.113 9.358 C 162.113 9.097 162.118 8.847 162.807 8.847 C 164.093 8.847 167.357 9.159 168.312 9.983 C 169.032 10.613 169.279 14.405 167.078 16.808 L 167.075 16.808 C 165.917 18.058 164.436 18.707 162.708 18.707 C 160.344 18.707 158.477 17.433 157.578 15.258 C 156.897 13.563 157.729 12.589 158.398 12.589 Z M 163.149 11.886 C 163.537 12.008 163.934 12.043 164.333 12.121 C 164.038 12.332 163.776 12.59 163.524 12.867 C 163.413 12.535 163.277 12.211 163.149 11.886 Z"
+fill="#505F79"
+    
+></path>
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each social icon in a `<li>` tag and make sure that the icons are all links using an `<a>` tag. Your final code should look similar to something like this:

`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`

- [Optional] I highly suggest you use either images or an icons to represent your social icons. Using either of these elements will allow your code to be more concise and easier to understand which will be a great help in the long run👍

Reply:
- [ ] As requested by the previous reviewer, kindly wrap your social links in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as `ul` tags are used to wrap most lists in a webpage

e.g
```
<ul>
<li><a><img/></a></li>
<li><a><img/></a></li>
<li><a><img/></a></li>
</ul>
```

Reply:
- [ ] As requested by the previous reviewer, kindly wrap your social links in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as `ul` tags are used to wrap most lists in a webpage
e.g
```HTML
<ul>
   <li><a><img/></a></li>
   <li><a><img/></a></li>
   <li><a><img/></a></li>
</ul>
```

- I would kindly advice you download the social media icon from the figma design and import them.into your project using an `<img>` tag rather than importing the icons using path.